README.md

```diff
@@ -108,6 +108,7 @@ Get pid informations.
 | Param | Type | Description |
 | --- | --- | --- |
 | pids | <code>Number</code> \| <code>Array.&lt;Number&gt;</code> \| <code>String</code> \| <code>Array.&lt;String&gt;</code> | A pid or a list of pids. |
+| options | <code>Object</code> | Currently used to set the history max age for wmic (`{maxage: 60000}`).  |
```

Comment:
Is really necessary to expose this to the users?
What are the use cases?

Reply:
Actually none but I thought it was a good idea do document it since it's available.

Reply:
Anyway see this:
https://github.com/simonepri/pidtree#pidtreepid-options-callback--promisearrayobject
to see a "correct" way to document default values